css/buttons.styl

```diff
@@ -44,6 +44,8 @@
 
 .pill-button
   @extends $reset-button
+  color: FOREGROUND
+  border-color: FOREGROUND
```

Comment:
Looks like colors are still being defined on the `pill-button` class here. I don't really want to go through the whole site where this class is being used to make sure there aren't unintended issues with this. Could you add the colors on the class where it's being used?